config.go

```diff
@@ -53,11 +53,14 @@ func NewWAFConfig() WAFConfig {
 
 // RequestBodyConfig controls access to the request body.
 type RequestBodyConfig interface {
-	// WithLimit sets the maximum number of bytes that can be read from the request body. Bytes beyond that set
-	// in WithInMemoryLimit will be buffered to disk.
+	// WithLimit sets the maximum number of bytes that can be read from the request body. Going beyond WithInMemoryLimit will
+	// result in buffering to disk.
+	// The settable upper limit for 32-bit machines is 2147483647 bytes (2GiB)
```

Comment:
I would discard this commit and add a link to a RATIONALE.md document explaining the implications of this. As a quick fix, I would just discard this line and tackle the deep explanation as part of https://github.com/corazawaf/coraza/issues/534